src/site/filters/liquid.js

```diff
@@ -12,6 +12,10 @@ function getPath(obj) {
 module.exports = function registerFilters() {
   const { cmsFeatureFlags } = global;
 
+  // Set timeout option to something higher (20mins)
+  // eslint-disable-next-line no-new
+  new liquid.Context({ timeout: 1200000 });
```

Comment:
As the node count increased, it took the templates too long to parse, so just increasing the timeout from the default  of 2 mins set in Tiny Liquid

Reply:
I'm familiar with this one 

```
Error: Timeout.
    at Timeout.<anonymous> (/vets-website/node_modules/tinyliquid/lib/index.js:79:16)
    at listOnTimeout (internal/timers.js:554:17)
    at processTimers (internal/timers.js:497:7)
```

it doesn't need to be assigned to anything? `liquid.context = new Context...`?

Reply:
I tried that initially and it was throwing errors, maybe just something buried in the plugin